tariff/fixed.go

```diff
@@ -21,6 +21,10 @@ func NewFixed(other map[string]interface{}) (*Fixed, error) {
 	return &cc, nil
 }
 
+func (t *Fixed) CurrentPrice() (float64, error) {
+	return t.Price, nil
+}
+
 func (t *Fixed) IsCheap() bool {
```

Comment:
Hier wär die Frage ob wir das `IsCheap` nicht lieber in die Site verschieben sollten.

Reply:
Ja, das wäre dann Teil des späteren Umbaus.